docs-gh-pages/running-a-node.md

```diff
@@ -33,38 +31,27 @@ A constellation node has two API's: A control API meant for the node operator to
+have firewalls blocking these ports, and often home networks do not have a stable ip address. For now, our official 
+recommendation is to use a cloud provider.
 
 ### Cloud hosting
-As a team we have the most expertise with Google Cloud, and it is our recommended platform for those without one already, but any cloud provider will work.
-On 
-[Google Cloud Platform](https://cloud.google.com/) 
-(GCP), choose *compute engine* using a Ubuntu or Debian machine. 
-You can also do the installations up on the cloud machine.
-Make sure the relevant ports are open / not firewalled. 
-
-#### Cloud providers
-Any cloud provider should work fine. While we do all our testing on GCP (Google Cloud Provider), we are not using any proprietary features. If you can launch a machine, install java 8 or higher, assign an external IP, and open the ports for the control API and peer API, you shouldn't have any problems.
+Walkthrough for [GCP](https://docs.google.com/document/u/0/d/1KkJSdydz2DsAlQKaVB8MVqqQuPMr-ggTB2Y2niKOodY/mobilebasic)
+Walkthrough for [AWS/DigitalOcean and Docker](https://docs.google.com/document/d/1qFYLqv2g_gHWWFs-x9YsPvrGzEocMPZS9hsJBegPrz0/edit)
 
 ## Manual Builds And Deployment
-The [build instructions](https://github.com/Constellation-Labs/constellation/blob/dev/docs/build-instructions.md) have more pointers on how to run a node.
+The [build instructions](https://github.com/Constellation-Labs/constellation/blob/dev/docs/build-instructions.md) 
+have more pointers on how to run a node. 
 
 ### Additional Tools
-Check out the 
-[recommended tools and frameworks](https://github.com/Constellation-Labs/constellation/blob/dev/docs/dependencies.md) 
-section in the docks. Here we highlight two:
-
-#### Docker
-We intend to have official docker images soon -- stay tuned.
+The DAG [Command Line Tool](https://github.com/zaemliss/Constellation)
 
-#### Terraform (Optional!)
-At Constellation Labs we use terraform to quickly launch and destroy clusters for testing. Our terraform configurations are checked in to the repository in the terraform directory, so they are available as a guide if you'd like to use it yourself. They automate some nice but optional things like running the node as a service. They are currently setup for GCP, and they have a few constellation-specific hardcoded variables (backend storage location, project name). That said -- if you're familiar with terraform, they should be straightforward to adapt for your uses. While we are not officially supporting this right now, we can provide some support on discord for this method.
+## Connecting To An Existing Network
+Be sure that you have the latest whitelist, which can be found by running 
+``wget https://github.com/Constellation-Labs/constellation/releases/latest/download/whitelisting``
+And you can update the whitelist with your info on [this spreadsheet](https://docs.google.com/spreadsheets/d/1MGBevI3MbhsN-oueC_q8ZPKRpWdPyaITcJpAhz60lPo/edit?pli=1#gid=0)
```

Comment:
One cannot update the whitelist manually because will not be able to join the cluster. Whitelisting file must be equal across the whole cluster

Reply:
Right right, but for new folks and for updating the whitelist for releases, we can have them update on the google sheet. Should we do differently?

Reply:
We started using Google Forms for the last round of testnet. I recommend we do the same moving forward for mainnet.